deploy/kustomize/base/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
```

Comment:
:+1: these are what I would think would be essential items for consumers to deploy HCO.